hack/lib/tracing.bash

```diff
@@ -232,16 +218,20 @@ function teardown_opentelemetry_tracing {
     oc delete -n openshift-operators "${csv}" --ignore-not-found
   fi
 
+  # Do not remove Jaeger if it's part of Service Mesh
   if [[ $(oc get crd servicemeshcontrolplanes.maistra.io --no-headers | wc -l) != 1 ]]; then
-    # Do not remove Jaeger if it's part of Service Mesh
-    oc delete -n "${TRACING_NAMESPACE}" jaeger.jaegertracing.io jaeger
-    timeout 600 "[[ \$(oc get -n ${TRACING_NAMESPACE} deployment jaeger --no-headers | wc -l) != 0 ]]"
+    if [[ $(oc get -n "${TRACING_NAMESPACE}" jaeger.jaegertracing.io jaeger --no-headers | wc -l) != 0 ]]; then
+      oc delete -n "${TRACING_NAMESPACE}" jaeger.jaegertracing.io jaeger --ignore-not-found
```

Comment:
the `ignore-not-found` is added because we previously deleted stuff even if they were not previously installed?

Reply:
I guess on this line we could remove the `--ignore-not-found` because we check one line above that it really exists. But it shouldn't harm. WDYT?

Reply:
Ok does not harm.